### PR TITLE
add translations to sidebar view headers, add qtbase translation loading

### PR DESCRIPTION
**Pull request by ewancg**
qtbase translation allows qmessagebox buttons ("standard buttons", as also seen in dialogs and stuff) to be translated. and probably a bunch of other stuff

translations to sidebar is self-explanatory; it looked like these translations were explicitly omitted (right next to instances where the title was translated just a few lines away, except as a tooltip). if this was intended, i'm not sure what the effect was meant to be but i think it should be translated either way. if not, oopsies

### CODE REVIEW

**Comment by ewancg** [comment]
just for the record, some of the tr calls added in the models try and translate the same string as the tooltip even though they have different purposes. for example, the delay effect header shows "Freq." but when hovered it becomes "Frequency"; but right next to it, there's "Ratio" which is just displayed as "Ratio" in the tooltip. maybe it would be worth adding a disambiguation; and if one is added, how would it disambiguate? by how it's meant to be interpreted (full word vs. abbreviation), or how it's shown (table header vs. tooltip)?

File: src/editor/sidemenu/UserListModel.cpp [hunk 1]
```diff
@@ -40,9 +40,9 @@ QVariant UserListModel::data(const QModelIndex &index, int role) const {
         return user->first;
       case UserListModel::Column::Ping:
         if (user->second.last_ping.has_value()) {
-          int ping = user->second.last_ping.value();
+          int ping = static_cast<int>(user->second.last_ping.value());
           if (ping > 1000) return QString("%1s").arg(ping / 1000);
```
Comment (OPNA2608) [inline src/editor/sidemenu/UserListModel.cpp:44]: Missed the string here.

File: src/editor/sidemenu/UserListModel.cpp [hunk 2]
```diff
@@ -40,9 +40,9 @@ QVariant UserListModel::data(const QModelIndex &index, int role) const {
...
-          return QString("%1ms").arg(ping);
+          return ping + tr("ms");
```
Comment (OPNA2608) [inline src/editor/sidemenu/UserListModel.cpp:45]: Does it even make sense to have these translated? I'd assume this specific part of the SI is adopted well enough that everyone either uses or at least understands `s` & `ms`.

If keeping it: Does it make sense to include the number in the string? I don't know if/how simple appending or the translation would interact with RTL languages.

```suggestion
          return tr("%1ms").arg(ping);
```

File: src/editor/sidemenu/OverdriveEffectModel.cpp [hunk 3]
```diff
@@ -97,11 +97,11 @@ QVariant OverdriveEffectModel::headerData(int section,
...
       switch (OverdriveEffectColumn(section)) {
         case OverdriveEffectColumn::Group:
-          return "G";
+          return tr("G");
         case OverdriveEffectColumn::Cut:
-          return "Cut";
+          return tr("Cut");
         case OverdriveEffectColumn::Amp:
-          return "Amp.";
+          return tr("Amp.");
```
Comment (yuxshao) [inline src/editor/sidemenu/OverdriveEffectModel.cpp:104]: Maybe add a comment about the context (section headers, thus should be small)?

File: src/editor/sidemenu/UserListModel.cpp (hunk 2 above)
Comment (yuxshao) [reply src/editor/sidemenu/UserListModel.cpp:45]: Yeah slight favour not to have it since s / ms seems p standard to me

File: src/main.cpp [hunk 4]
```diff
@@ -54,7 +55,7 @@ int main(int argc, char *argv[]) {
   a->setOrganizationDomain("ptweb.me");
   a->setApplicationName("ptcollab");
   a->setApplicationVersion(Settings::Version::string());
-  QTranslator translator;
+  QTranslator translator, /*qtTranslator,*/ baseTranslator;
```
Comment (yuxshao) [inline src/main.cpp:58]: (nit: Seems like this stray comment is unnecessary?)

**Review by yuxshao — COMMENTED** [review]
Just a few tiny things

File: src/main.cpp (hunk 4 above)
Comment (ewancg) [reply src/main.cpp:58]: Look below

**Pull request merged**